### ISSUE TITLE
Mumble SSL startup errors

### ISSUE THREAD

**Issue by felixsinger** [mumble-voip/mumble#3061]
Just compiled and installed Mumble from this AUR package https://aur.archlinux.org/packages/mumble-snapshot-minimal/

While starting Mumble I get those errors
```
Found multiple libssl.so copies in binary: /usr/lib/libssl.so.1.1, /usr/lib/libssl.so.1.0.0
Found multiple libcrypto.so copies in binary: /usr/lib/libcrypto.so.1.1, /usr/lib/libcrypto.so.1.0.0
Aborting due to previous errors
Aborted (core dumped)
```

**Comment by Athemis** [comment]
What you are hitting is #2621. The actual problem is that the PKGBUILD is broken:
Since Archlinux switched to openssl-1.1.0, qt5 is also built against openssl-1.1.0. But mumble currently cannot be successfully compiled against this version of openssl (see #3053). That's why the author of the AUR package you used forced compilation against the legacy version of openssl. But this results in a broken binary because it is simultaneously linked against a legacy openssl version and a qt5 version that uses another version of openssl. As stated in #2621, this is unsupported. Hence, the AUR package is broken. IMHO you have two choices: 

1. Wait until #3053 is fixed and compile against openssl-1.1.0
2. Use mumble from Archlinux's extra repos (which is version 1.2.19 and compiled against qt4 which still is linked against legacy openssl)

**Comment by felixsinger** [comment]
Fixed by additional build parameter :)
